compiler/rustc_codegen_llvm/src/llvm/ffi.rs

```diff
@@ -1772,6 +1772,59 @@ unsafe extern "C" {
         Scope: &'ll Metadata,
         InlinedAt: Option<&'ll Metadata>,
     ) -> &'ll Metadata;
+
+    pub(crate) fn LLVMDIBuilderCreateSubroutineType<'ll>(
```

Comment:
trivially obvious but this can in fact do a UB if we violate it 
```suggestion
    /// create debuginfo for subroutine types
    /// # Safety
    /// - `ParameterTypes` and `NumParameterTypes` must be from the same slice
    pub(crate) fn LLVMDIBuilderCreateSubroutineType<'ll>(
```